scripts/controlnet.py

```diff
@@ -119,9 +119,9 @@ def image_dict_from_unit(unit) -> Optional[Dict[str, np.ndarray]]:
     if isinstance(result['image'], str):
         result['image'] = external_code.to_base64_nparray(result['image'])
 
-    if isinstance(result['mask'], str):
+    if isinstance(result['mask'], str) and result['mask']:
         result['mask'] = external_code.to_base64_nparray(result['mask'])
-    elif result['mask'] is None:
+    else:
         result['mask'] = np.zeros_like(result['image'], dtype=np.uint8)
```

Comment:
If the mask is already an np.array, it will get replaced here with zeros. Consider leaving the check for None.

Reply:
ooo yeah.. on it!